src/main/java/frc/robot/commands/LimelightAlign.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.LimelightSubsystem;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class LimelightAlign extends CommandBase {
+  private LimelightSubsystem m_LimelightSubsystem;
+
+  /** Creates a new ShooterPrimary. */
+  public LimelightAlign(LimelightSubsystem LimelightSubsystem) {
+    addRequirements(LimelightSubsystem);
```

Comment:
Talking with @Majorferguson21 it's unsure to me if you need to add drivebase as a requirement here.

 @azvanderpas thoughts?